### ISSUE TITLE
`.ps.gz` files handled by postscript backend.

### ISSUE THREAD

**Issue by btimby** [btimby/fulltext#55]
The postscript backend uses pstotext, which cannot handle `.ps.gz` files. These should be sent to the gzip backend first.

```
ShellError: The command `pstotext /media/mounts/10.100.31.220.America-INC/Engineering/talk.ps.gz` failed with exit code 3
------------- stdout -------------
b''------------- stderr -------------
b'GPL Ghostscript 9.06: Unrecoverable error, exit code 1\n'
  File "audittrail/workers/fingerprinter.py", line 235, in fingerprinter
    fp, md5, size = file_location.fingerprint_file()
  File "audittrail/workers/fingerprinter.py", line 66, in fingerprint_file
    first = next(fingerprints)
  File "duster/__init__.py", line 282, in dust
    for fn, type, size, ct, mt, raw, text in unwrap(path):
  File "duster/containers.py", line 228, in unwrap
    for item in unwrap_text(path, type):
  File "duster/containers.py", line 145, in unwrap_text
    text = fulltext.get(path, mime=type, kwargs=kwargs)
  File "fulltext/__init__.py", line 374, in get
    backend=backend, kwargs=kwargs)
  File "fulltext/__init__.py", line 351, in _get
    text = fun(backend_mod, path_or_file, **kwargs)
  File "fulltext/__init__.py", line 230, in _get_path
    return backend._get_path(path, **kwargs)
  File "fulltext/backends/__ps.py", line 24, in _get_path
    return run('pstotext', path).decode('utf8')
  File "fulltext/util.py", line 108, in run
    raise ShellError(' '.join(cmd), pipe.returncode, stdout, stderr)```

**Comment by giampaolo** [comment]
This is a duplicate of #56 and it is fixed by #57.

**Comment by giampaolo** [comment]
Closing as fixed.
